bip-0141.mediawiki

```diff
@@ -196,7 +196,7 @@ Comparing with the previous example, the scriptPubKey is 1 byte bigger and the s
 
 The following example is an 1-of-2 multi-signature version 0 pay-to-witness-script-hash (P2WSH).
 
-    witness:      0 <signature1> <1 <pubkey1> <pubkey2> 2 CHECKMULTISIG>
+    witness:      0 <signature1> 1 <pubkey1> <pubkey2> 2 CHECKMULTISIG>
```

Comment:
There was no typo to begin with

Reply:
Nevermind, I misread this. was about to close.